haskell/adventure.hs

```diff
@@ -0,0 +1,426 @@
+-- Przygody Gerwanta z Riviery, by Drygaś Filip, Lew Filip, Lipniacki Daniel.
+{-# LANGUAGE BlockArguments #-}
+{-# LANGUAGE OverloadedLists #-}
+
+import Control.Monad.State
+import Data.List (intercalate)
+import Data.Map.Strict (Map)
+import qualified Data.Map.Strict as Map
+import Data.Maybe (fromJust, fromMaybe)
+import Data.Set (Set)
+import qualified Data.Set as Set
+import System.IO
+import Data.Set (toList)
+
+-- game content
+
+introductionText =
+  [ "Gerwant z Riviery, szkoły nosacza, podróżuje szlakami Królestw Północy już wiele dni.",
+    "Towarzyszy mu jedynie deszcz i jego wierny koń Piwonia."
+  ]
+
+instructionsText =
+  [ "Dostępne polecenia:",
+    "polecenia              -- by wyświetlić listę poleceń.",
+    "idź <kierunek>         -- by pójść w danym kierunku.",
+    "podnieś <przedmiot>    -- by podnieść przedmiot.",
+    "upuść <przedmiot>      -- by upuścić przedmiot.",
+    "ekwipunek              -- by sprawdzić ekwipunek.",
+    "obejrzyj <przedmiot>   -- by obejrzeć przedmiot.",
+    "rozmawiaj <npc>        -- by porozmawiać z NPC.",
+    "spytaj <npc> <coś>     -- by spytać NPC o coś.",
+    "rozejrzyj się          -- by dowiedzieć się, gdzie jesteś.",
+    "przygotuj się do walki -- by odgadnąć rodzaj potwora i zwiększyć swoje szanse na wygraną.",
```

Comment:
This should be hidden and revealed when "Gerwant" steps into the "Pieczara" location

Reply:
Removed from help, will not add it when revleaed

Reply:
I ment revealed in "Pieczara" location description, which is already in

Reply:
this is the case now